### DISCUSSION TITLE
اینستاگرام‌ من بدون فیلتر است

### DISCUSSION THREAD

**Discussion by FarhadiAlireza** [GFW-knocker/gfw_resist_tls_proxy#81] [Ideas]
سلام وقت بخیر و خسته نباشید به شما سروران

اینستاگرام بنده بدون وی پی ان اجرا میشود 

تست آی پی گرفتم ( آی پی ام ایران بود )
هیچ پروکسی هم متصل نیست 

به مودم هم وصل نیستم ، با نت ایرانسل وصل میشم

خلاصه همه چی رو تست گرفتم ، انگاری واقعا بدون وی پی ان وارد ایسنتاگرام میشم 😄😁

ولی برای بقیه اعضا خانواده فیلتر هست

نمیدونم چطوری داره کار میکنه فک کنم یه باگی هست داخل فیلترینگ ، خواستم به اطلاعتون برسونم 

اگه تستی ، لاگی یا هر چیزی که بهتون کمک کنه واستون انجام بدم برای دور زدن فیلترینگ اینستاگرام 

سپاس فراوان

**Comment by GFW-knocker** [comment]
فیلترینگ لحظه ای ممکنه قطع شه
اگه همیشه براتون وصل هست
 ایپی و دامنه اش رو دربیارید 
اپ های نتورک مانیتور برا اندروید هست

**Comment by FarhadiAlireza** [comment]
آیپی و دامنه ی چی رو در بیارم و امروز متوجه این شدم که مال خوده گوشیم هستش و با هر نتی بدون وی پی ان وصل میشه ! 
و اگه ممکنه یه اپ واسه نتورک مانیتور بگید ، برم تست کنم 
بسیار ممنون

**Reply by GFW-knocker** [reply]
netGuard
android app - no root needed
see which ip and domain ,  instagram connected to

**Comment by TKZ700** [comment]
> سلام وقت بخیر و خسته نباشید به شما سروران
> 
> اینستاگرام بنده بدون وی پی ان اجرا میشود 
> 
> تست آی پی گرفتم ( آی پی ام ایران بود )
> هیچ پروکسی هم متصل نیست 
> 
> به مودم هم وصل نیستم ، با نت ایرانسل وصل میشم
> 
> خلاصه همه چی رو تست گرفتم ، انگاری واقعا بدون وی پی ان وارد ایسنتاگرام میشم 😄😁
> 
> ولی برای بقیه اعضا خانواده فیلتر هست
> 
> نمیدونم چطوری داره کار میکنه فک کنم یه باگی هست داخل فیلترینگ ، خواستم به اطلاعتون برسونم 
> 
> اگه تستی ، لاگی یا هر چیزی که بهتون کمک کنه واستون انجام بدم برای دور زدن فیلترینگ اینستاگرام 
> 
> سپاس فراوان

برا من یه مدت یوتیوب فیلتر نبود ولی الان دیگه اونطور نیست🥲
